src/components/StateBrowserNetwork.vue

```diff
@@ -438,11 +503,24 @@ export default {
     font-size: 0.9em;
 }
 
-.kiwi-statebrowser-channels-options {
+.kiwi-statebrowser-buffers-options {
     text-align: left;
+    margin-left: 5px;
+}
+
+.kiwi-statebrowser-channels-header-label,
+.kiwi-statebrowser-queries-header-label {
+    display: inline-block;
+    font-size: 0.8em;
+    text-transform: uppercase;
+    margin-left: 5px;
+}
+
+.kiwi-statebrowser-buffers-toggle {
+    width: 10px;
 }
 
-.kiwi-statebrowser-channels-option {
```

Comment:
This change is technically correct and this class was badly named originally. But this will need all themes updating and would break any third party themes too - a much bigger task so we've left this alone for now.

Reply:
OK rolled that back.